openjdk/playlist.xml

```diff
@@ -1657,6 +1657,42 @@
 			<feature>FIPS140_3_OpenJCEPlusFIPS.FIPS140-3:nonapplicable</feature>
 		</features>
 	</test>
+	<test>
+		<testCaseName>jdk_restricted_security_j9</testCaseName>
+		<disables>
+			<disable>
+				<comment>only applicable for Semeru atm</comment>
+				<vendor>eclipse</vendor>
+			</disable>
+			<disable>
+				<comment>Not applicable on zos atm</comment>
+				<platform>.*zos.*</platform>
+			</disable>
+		</disables>
```

Comment:
according to https://github.com/adoptium/aqa-tests/pull/5782#issuecomment-2520549828, please remove all disables. And also remove line 1693 `<impl>ibm</impl>`